Update charger_state to use native zaptec values to support hacs v0.8

steinmn opened the pull request:
The Zaptec Custom component for HA v0.8 will migrate to use the zaptec native names for the charger state (see https://github.com/custom-components/zaptec/pull/215 for details).

This should not be merged until 0.8.0 is released

steinmn committed:
Update charger_state to use native zaptec values to support hacs v0.8

The Zaptec Custom component for HA v0.8 will migrate to use the zaptec native names for the charger state (see https://github.com/custom-components/zaptec/pull/215 for details.

This should not be merged until 0.8.0 is released

steinmn committed:
Use lower case for charger state values

svenakela commented:
Cool, thanks for the heads up. Have some features being added, will incorporate this for sure.

steinmn committed:
Switch to using number.set_value instead of zaptec.limit_current

zaptec.limit_current is broken in 0.8.1, but number.set_value on available_current has the same functionality

steinmn commented:
@svenakela v0.8.1 is now out, but the `zaptec.limit_current`-service is currently broken (see https://github.com/custom-components/zaptec/issues/283). The `number.set_value` service on `available_current` should cover the same functionality, so I've attempted to switch to that. Up to you if you want to test it like this or wait for 0.8.2 and revert the last commit to keep using `zaptec.limit_current`.

sveinse reviewed (commented):
LGTM - but I'm no expert in blueprints, so I'm happy if someone else also chips in to verify.

sveinse commented:
Releasing v0.8.2 seems urgent, but we should have this included. Can someone please verify the new blueprint, please?

sveinse commented:
~~@steinmn can you update the docs for the PR too please? I think the README needs a note in the "what's changed" and or "breaking changes" for this. I think people need to be advised that their previous blueprint is broken. Seems to be a few using this.~~

**edit**: That was not for this repo. Bummer.

I'm a bit afk today, but I'll be available later this evening and tomorrow.

sveinse commented:
The blueprint is (unfortunately) language dependent see https://github.com/custom-components/zaptec/issues/286#issuecomment-3193693474, so the README should probably contain a guide on how to assist the users how to modify the blueprint to work for them.

steinmn committed:
Revert "Switch to using number.set_value instead of zaptec.limit_current"

This reverts commit 10aedc9fd4843b305174ad9119e42d57023b34c3.

steinmn committed:
Update readme with minimum zaptec version and language requirement

steinmn commented:
v0.8.2 is out with a fix for the `zaptec.limit_current`-service, so I've reverted the switch to `number.set_value` and added a bit on the language topic to the Requirements section of the readme.

svenakela commented:
I'm not even sure which comments are for me anymore... :laughing:

steinmn commented:
> I'm not even sure which comments are for me anymore... 😆

:rofl: sorry about that @svenakela, sveinse and I have had many discussions across many issues and PRs over the past months, so it's easy to lose track sometimes.

Bottom line: v0.8.2 of the zaptec integration is officially released and seemingly stable, so would be great if you could review and test this out :+1:
